src/components/HamburgerMenu/__test__/SidebarNav.test.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import SidebarNav from '../SidebarNav';
+import renderer from 'react-test-renderer';
+test('Humburger menu snapshot testd succesfuly', () => {
+  const tree = renderer.create(<SidebarNav />).toJSON();
+  expect(tree).toMatchSnapshot();
```

Comment:
snapshot tests are for final version, otherwise, you will keep creating them for every change and it doesn't make sense